client/state/data-layer/wpcom/sites/comments/test/index.js

```diff
@@ -29,10 +29,10 @@ describe( '#addComments', () => {
 
 	beforeEach( () => ( dispatch = spy() ) );
 
-	it( 'should dispatch no actions for no comments', () => {
+	it( 'should dispatch one action for no comments', () => {
```

Comment:
maybe this isn't so descriptive. why the change form none to one?

how about instead we test for a positive _and_ descriptive behavior, like `it( 'should dispatch a tree initialization action' )` (of course, pending the value of that action being dispatched)